### DISCUSSION TITLE
The combobox doesn't detect that is located at the page bottom.

### DISCUSSION THREAD

**Discussion by roynert** [twilio-labs/paste#3061] [General] [answered]
I had to add the combobox on the page bottom. When the combobox displays the list of options, the users do not see the list because it displays below the end of the page. Then, the user has to scroll down to see the list.
Is there a configuration that allows the combobox displays the options list above the combo element?

<img width="399" alt="image" src="https://user-images.githubusercontent.com/122480559/222004176-c4fd140b-b866-4dc8-8f04-a912c8017e36.png">

**Comment by serifluous** [comment, marked as answer]
Thanks for reporting this, @roynert ! I'll follow up with our team and get back to you.

Would you be able to share a [CodeSandbox reproduction ](https://codesandbox.io/s/paste-bug-report-j8jxpf)of the issue in the meantime?

**Reply by serifluous** [reply]
Hey @roynert , please let us know if you're able to reproduce this in CodeSandbox! I'll mark this discussion as closed right now due to inactivity, but we'd be more than happy to open it back up with any follow-up.
